internal/querycoordv2/meta/target_manager.go

```diff
@@ -0,0 +1,165 @@
+package meta
+
+import (
+	"sync"
+
+	"github.com/milvus-io/milvus/internal/proto/datapb"
+	"github.com/milvus-io/milvus/internal/util/funcutil"
+)
+
+type TargetManager struct {
```

Comment:
TargetManager definitely need more logs.
We need to know what segments, dm channels is added